relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -331,6 +334,17 @@ pub(crate) fn extract_tags(
             span_tags.insert(SpanTagKey::Group, span_group);
 
             let truncated = truncate_string(scrubbed_desc, config.max_tag_value_size);
+            if span_op.starts_with("resource.") {
+                if let Some(ext) = truncated
+                    .rsplit('/')
+                    .next()
+                    .and_then(|last_segment| last_segment.rsplit_once('.'))
+                    .map(|(_, extension)| extension)
+                {
+                    span_tags.insert(SpanTagKey::Extension, ext.to_owned());
+                }
```

Comment:
Just to clarify here, if there is no extension, we simply do not insert it correct? This would allow to query for spans that don't have an extension